spec/features/collection_drafts/gkr_logging_spec.rb

```diff
@@ -7,30 +7,38 @@
 
   context 'when saving GKR recommendations' do
     before do
-      allow_any_instance_of(GKRKeywordRecommendations).to receive(:fetch_keyword_recommendations).and_return({ id: 22, recommendations: ['EARTH SCIENCE > ATMOSPHERE > ATMOSPHERIC CHEMISTRY > OXYGEN COMPOUNDS > OZONE', 'EARTH SCIENCE > ATMOSPHERE > AIR QUALITY'] })
+      allow_any_instance_of(GKRKeywordRecommendations).to receive(:fetch_keyword_recommendations).and_return({ id: 22,
+                                                                                                               recommendations: ['EARTH SCIENCE > ATMOSPHERE > ATMOSPHERIC CHEMISTRY > OXYGEN COMPOUNDS > OZONE', 'EARTH SCIENCE > ATMOSPHERE > AIR QUALITY'],
+                                                                                                               uuids: { 'EARTH SCIENCE > ATMOSPHERE > ATMOSPHERIC CHEMISTRY > OXYGEN COMPOUNDS > OZONE': 1, 'EARTH SCIENCE > ATMOSPHERE > AIR QUALITY': 2 } })
       visit edit_collection_draft_path(@draft, form: 'descriptive_keywords')
     end
 
     it 'logs the Save operation' do
       click_on 'Expand All'
-      allow(Rails.logger).to receive(:info)
-      expect(Rails.logger).to receive(:info).with(start_with('GkrLog: type: SAVE'))
       within '.nav-top' do
-        click_on 'Save'
+        VCR.use_cassette('gkr/send_feedback', record: :none) do
```

Comment:
You opted to keep this?

Reply:
Yes I did as the results are a bit different.  The gkr_logging_spec only requires the PUT call. The other spec files require a POST and the PUT. gkr_logging_spec did not have 2 VCR's associated with it. Only the send_feedback. I tried combining the send_feedback.yml VCR PUT call into the  initial_keyword_recommendations.yml VCR file, but I was getting some errors as well.

Let me know if you feel strongly about combining, and I can look into it further.